tests/ha/yast_cluster_init.pm

```diff
@@ -68,17 +77,28 @@ sub run {
     wait_screen_change { send_key 'alt-n' };
 
     # Cluster service settings screen
+    # This screen is different depending on the yast2-cluster version.
+    # In 15-SP2 Alt-E is required to enable the cluster at boot time,
+    # and Alt-a to start cluster now. In older versions, Alt-a is
+    # required to start the cluster on boot and Alt-S to start the
+    # cluster now. The following combination of send_key calls tries
+    # to cover most cases
     assert_screen 'yast-cluster-service';
-    send_key 'alt-a';    # Start pacemaker during boot
+    send_key 'alt-E';    # Enable cluster
```

Comment:
If it's only on 15-SP2, is not better to add a `if sle('15-sp2')`?

Reply:
In one of the versions I did that, but then found a way to work all 3 three shortcuts/hotkeys so that it works in the current and old screen. IMO it's better this way, in case the new version of `yast2-cluster` is pushed to older systems via an MU.